firefox-ios/Client/Frontend/TabContentsScripts/LoginsHelper.swift

```diff
@@ -160,7 +160,7 @@ class LoginsHelper: @unchecked Sendable, TabContentScript, FeatureFlaggable {
                     windowUUID: tab.windowUUID,
                     actionType: GeneralBrowserActionType.showPasswordGenerator)
 
-                store.dispatchLegacy(newAction)
```

Comment:
FYI [`usercontentcontroller(_:didreceive:)`](https://developer.apple.com/documentation/webkit/wkscriptmessagehandler/usercontentcontroller(_:didreceive:)) method is `@MainActor`